src/main/java/com/pinterest/secor/parser/MessageParser.java

```diff
@@ -48,13 +52,22 @@ public MessageParser(SecorConfig config) {
             String separatorPattern = Pattern.quote(mConfig.getMessageTimestampNameSeparator());
             mNestedFields = mConfig.getMessageTimestampName().split(separatorPattern);
         }
+        try {
+            messageTransformer = ReflectionUtil
+                .createMessageTransformer(mConfig.getMessageTransformerClass(), mConfig);
+        } catch (Exception e) {
+            LOG.warn("Unable to initialize custom message transformer class... Using default", e);
```

Comment:
If I throw the RuntimeException from constructor then it'll lead to modify all the message parsers for the same. Is it fine?